### PR TITLE
People will now bleed more if dragged

### PR DESCRIPTION
**Pull request by ghost**
Ported from /vg/
No they won't bleed if in a locker/crate or on a ROLLER BED. (they may bump their head in crates, however!). They bleed even more in crit. However it's ok in 0-gravity.

:cl: More Robust Than You
add: People will now bleed more if dragged. START USING THE FUCKING ROLLER BEDS!
/:cl:

### CODE REVIEW

**Comment by cedarbridge** [comment]
Definitely a good move for making medbay more involved.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -493,8 +493,25 @@
...
+	
+	if(bleed_rate && !buckled && has_gravity(loc) && lying)
+		if(prob(2) && istype(loc, /obj/structure/closet))
+			visible_message("<span class='warning'>[src] bumps their head against \the [src]!</span>")
+			apply_damage(rand(1, 2.5), BRUTE, "head")
+		else
+			if(prob(getBruteLoss() / 5) && health > HEALTH_THRESHOLD_CRIT)
+				bleed(rand(bleed_rate*0.75, bleed_rate))
+				add_splatter_floor(get_turf(src))
+				visible_message("<span class='warning'>\The [src] loses some blood from being dragged!</span>")
```
Comment (deathride58) [inline code/modules/mob/living/living.dm]: The message here is a little weird. Perhaps something like "[src]'s wounds are torn wider from being dragged across the floor!" would work better?

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -493,8 +493,25 @@
 		s_active.close(src)
 
 	if(lying && !buckled && prob(getBruteLoss()*200/maxHealth))
-
 		makeTrail(newloc, T, old_direction)
```
Comment (ChangelingRain) [inline code/modules/mob/living/living.dm]: you already lose blood by being dragged via this proc.

Comment (MoreRobustThanYou) [reply code/modules/mob/living/living.dm]: I can't find it.

Comment (MoreRobustThanYou) [reply code/modules/mob/living/living.dm]: makeTrail doesn't actually make you lose blood

Comment (ChangelingRain) [reply code/modules/mob/living/living.dm]: sure it does.

Comment (MoreRobustThanYou) [reply code/modules/mob/living/living.dm]: Well i'll remove that!

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 3]
```diff
@@ -58,6 +58,21 @@
 				//End bloody footprints
 
 				S.step_action()
+		
+	if(bleed_rate && !buckled && has_gravity(loc) && lying)
+		if(prob(2) && istype(loc, /obj/structure/closet))
+			visible_message("<span class='warning'>[src] bumps their head against \the [src]!</span>")
```
Comment (RandomMarine) [inline code/modules/mob/living/carbon/human/human_movement.dm]: You should probably check if they have a head in the first place.

Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/human_movement.dm]: You can only bump your head if you're bleeding?

File: code/modules/mob/living/living.dm (hunk 2 above)
Comment (PKPenguin321) [reply code/modules/mob/living/living.dm]: What's the point of that?

File: code/modules/mob/living/carbon/human/human_movement.dm (hunk 3 above)
Comment (RandomMarine) [reply code/modules/mob/living/carbon/human/human_movement.dm]: In addition, does this message reveal the identity of the lockered person for witnesses?
If it does, that's not something they should be able to know.

Comment (XDTM) [reply code/modules/mob/living/carbon/human/human_movement.dm]: If traumas are merged this should totally have a chance of concussing him

**Comment by Jalleo** [comment]
Its probably better to remake the PR but make it more damaging and provide feedback to the dragger every so often that it is damaging the person being dragged.

**Pull request closed without merging**